starter-code/routes/authRoutes.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
+const bcrypt = require("bcrypt");
+const User = require("../models/Users");
+const saltRounds = 10;
+const router = express.Router();
+
+router.get("/", (req, res, next) => {
+  res.render("index");
+});
+
+router.get("/signup", (req, res, next) => {
+  res.render("../views/signup.hbs");
```

Comment:
is not need it to writ the `.hbs`extension. Node know that you're using hbs has view's template. It's already defined in the `app.js`